client/src/unifyfs.c

```diff
@@ -1059,39 +1058,28 @@ int unifyfs_gfid_read_reqs(read_req_t* in_reqs, int in_count)
     /* we select different rpcs depending on the number of
      * read requests */
     if (count > 1) {
-        /* got multiple read requests,
-         * build up a flat buffer to include them all */
-        flatcc_builder_t builder;
-        flatcc_builder_init(&builder);
-
-        /* create request vector */
-        unifyfs_Extent_vec_start(&builder);
-
-        /* fill in values for each request entry */
+        /* got multiple read requests */
+        size_t size = (size_t)count * sizeof(unifyfs_extent_t);
+        void* buffer = malloc(size);
```

Comment:
You removed the `free(buffer);` below on line 1094. Still need it for this?

Reply:
Good catch - it's still needed.

Reply:
for flatcc.m4, just missed it when removing files

Reply:
I just pushed updates for these two oversights.